### ISSUE TITLE
Sending a message should jump to the bottom of the channel

### ISSUE THREAD

**Issue by viasux** [ajbura/cinny#255]
#### Describe the bug
This appears to only happen sometimes, unsure what makes it not happen

#### To Reproduce
1. Go to a chat
2. Scroll up
3. Type something & send
4. Doesn't scroll down

#### Expected behavior
I'd expect it to jump to the bottom of the chat, so I could see a response to my message, and, sometimes it does appear to do this.

#### Desktop (please complete the following information):
 - OS: [e.g. Windows, MacOS] Windows 10
 - Browser: [e.g. chrome, firefox] Libre Wolf
 - Version: [e.g. 3.22] 1.6.1
 - Matrix homeserver: [e.g. matrix.org] yiff.social

**Comment by tezlm** [comment]
This is actually very annoying. Sometimes I am replying to messages and get sent to the bottom of the timeline and have to scroll back up. It would be better to not automatically jump, and instead implement a keybinding (eg. <kbd>esc</kbd>) to jump to the bottom.

**Comment by viasux** [comment]
I also think that sending a message should mark the channel as read.

**Comment by deanveloper** [comment]
i agree with @tezlm. pressing ESC should scroll to the bottom, and mark the space as read

**Comment by Miepee** [comment]
Personally agree with tezlm here. 
I can however see that some people would like that behavior. Having this as an option in the appearance setting could maybe please both parties? Those who like it, can toggle it on, those who don't leave it disabled.

**Comment by jivanpal** [comment]
I'm in favour of a toggle for this. FWIW, Element Web/Desktop already has one.

**Comment by jman-schief** [comment]
This also probably applies when _receiving_ a message in the chat. I expect:

1. Go to a chat
2. scroll to the last message
4. wait for a new message to be posted
4. Chat window automatically scrolls down so I can see the new message in full

As new messages arrive, I expect the view to scroll accordingly.

**Comment by ilar** [comment]
I too expect the chat to show latest messages. In high-activity channels on my other monitor, I have zero interest in constantly flipping over to some other browser window, on a completely separate monitor, just to keep up with the chat.

IMO, have it scroll on new messages if you're already at the end of what was previously the last message, and if people complain, consider adding a toggle for those who want it to stay in place. Scroll on new message has been the messaging standard since the late '80s.

**Comment by kfiven** [comment]
If you are at the bottom of chat then Cinny does automatically scroll, this issue is about if you are scrolled up and send a message, whether it should scroll down or stays there.

@jman-schief and @ilar What you both commented is already supported.

**Comment by ilar** [comment]
It is definitely not- I've spent all day missing bits of conversation multiple times because it's not scrolling down when people talk.

**Comment by kfiven** [comment]
It got fixed in https://github.com/cinnyapp/cinny/issues/1500 recently so I request you to try again.
Please stop mixing another issue in this, as explained this is a different feature request and if you still do not find it working open new issue.
